src/main/java/com/aws/greengrass/clientdevices/auth/api/UseCases.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.api;
+
+import com.aws.greengrass.config.Topics;
+import com.aws.greengrass.dependency.Context;
+
+import javax.inject.Inject;
+
+
+public class UseCases {
```

Comment:
Just a wrapper that allow us to select a useCase class and fire it from a place where we want to call a "controller"